src/ImageBuilder/custom/New-AzImageBuilderTemplate/New-AzImageBuilderTemplate_CreateExpanded.ps1

```diff
@@ -195,21 +195,23 @@ param(
     # To construct, see NOTES section for DISTRIBUTE properties and create a hash table.
     ${Distribute},
 
-    [Parameter(Mandatory)]
-    [ArgumentCompleter([Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Support.ResourceIdentityType])]
-    [Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Category('Body')]
-    [Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Support.ResourceIdentityType]
-    # The type of identity used for the image template.
-    # The type 'None' will remove any identities from the image template.
-    ${IdentityType},
+    # Hide as IdentityType only can be 'UserAssigned'
+    # 'None' is not supported as removing identity is not supported when creating or updating an image template."
+    # [Parameter(Mandatory)]
+    # [ArgumentCompleter([Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Support.ResourceIdentityType])]
+    # [Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Category('Body')]
+    # [Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Support.ResourceIdentityType]
+    # # The type of identity used for the image template.
+    # # The type 'None' will remove any identities from the image template.
+    # ${IdentityType},
 
     [Parameter(Mandatory)]
     [Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Category('Body')]
     [Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Runtime.Info(PossibleTypes=([Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Models.Api20220214.IImageTemplateIdentityUserAssignedIdentities]))]
-    [System.Collections.Hashtable]
+    [System.String]
     # The list of user identities associated with the image template.
     # The user identity dictionary key references will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}'.
-    ${UserAssignedIdentity},
+    ${UserAssignedIdentityId},
```

Comment:
This parameter should be a `string[]` instead of `string`

Reply:
In previous version, it's string. https://docs.microsoft.com/en-us/powershell/module/az.imagebuilder/New-AzImageBuilderTemplate?view=azps-8.2.0#syntax. No intention to modify it so far.

Reply:
Sounds good. Pleazse approve this PR